1-JavaScript/Week3/assignment/ex2-mondaysWorth.test.js

```diff
@@ -30,8 +31,13 @@ const mondayTasks = [
 
 const hourlyRate = 25;
 
-function computeEarnings(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function computeEarnings(task, hourlyRate) {
+const earnings=task 
+.map(task=>(task.duration /60)* hourlyRate)
+.reduce ((total,current)=> total + current,0);
```

Comment:
```suggestion
const earnings = task
  .map((task) => (task.duration / 60) * hourlyRate)
  .reduce((total, current) => total + current, 0);
```

"It's already good, but I prefer to have a space between the symbols for better readability. Here’s how it looks:
;
Since we have `prettier` configured in `package.json`, you can run `npm install` to ensure it works automatically and formats the code.